docs/pages/identity-security/identity-security.mdx

```diff
@@ -2,46 +2,148 @@
+  [connections](./policy-connections.mdx) (identities, user group & actions)
 
-Identity Security is a separately licensed product and is available to Teleport Enterprise customers.
-Access Graph is a capability of Identity Security that displays the relationships of
-policies of users, groups, and computing resources.
+  **[View Standing Privileges](./how-to-use.mdx#dashboard-standing-privileges)**  
+  **[Set up Crown Jewels](./crown-jewels.mdx)**  
+  **[Manage Audit Events](./integrations/teleport.mdx)**
+</TextWithMedia>
 
-To verify the availability of the Access Graph, ensure that the Teleport Identity Security icon is present in the navigation sidebar.
-
-<Admonition type="note">
-Note: For managed Enterprise customers, Identity Security is enabled by default.
-If you are a self-hosted Teleport customer, you will need to [deploy the Access Graph Service](access-graph/access-graph.mdx) and ensure you have an updated
-`license.pem` with Identity Security enabled to use it.
-</Admonition>
-
-## Identity Activity Center
-
-(!docs/pages/includes/policy/identity-activity-center.mdx!)
-
-### Identity Activity Center guides and supported integrations
-
-- [Enable Identity Activity Center](access-graph/identity-activity-center.mdx)
-- [Discover AWS Access Patterns and CloudTrail events with Teleport Identity Security](./integrations/aws-sync.mdx) guide.
-- [Analyze and correlate Teleport Audit Logs](./integrations/teleport.mdx)
-- [Analyze and correlate GitHub Audit Logs and Access Paths](./integrations/github.mdx) 
-- [Analyze and correlate Okta Audit Logs and Access Paths](../identity-governance/okta/okta.mdx)
-
-## Identity Security guides
-
-<DocCardList />
+<Integrations
+  title="Integration"
+  integrations={[
+    {
+      title: 'GitHub',
+      description: 'Connect GitHub and stream audit logs, map permissions, and track user behavior to stay ahead of access issues.',
+      href: './integrations/github/',
+      iconColor: '#0000001A',
+      iconComponent: githubSvg,
+    },
+    {
+      title: 'Okta',
+      description: 'Integrate your Okta account to visualize your access policies, user roles, and group memberships.',
+      href: '../identity-governance/okta/',
+      iconColor: '#0000001A',
+      iconComponent: oktaSvg,
+    },
+    {
+      title: 'AWS',
+      description: 'Leverage the power of Access Graph by connecting AWS account to visualize IAM roles, policies, and access relationships.',
+      href: './integrations/aws-sync/',
+      iconColor: '#E87B0C1A',
+      iconComponent: awsSvg,
+    },
+    {
+      title: 'Teleport',
+      description: 'Connect Teleport audit logs to trace the full identity chain from authentication through resource access, enabling ...',
+      href: './integrations/teleport/',
+      iconColor: '#512FC91A',
+      iconComponent: teleportSvg,
+    },
+  ]}
+  mainLink={
+    {
+      title: "View all Integrations",
```

Comment:
It took me a while to find the button to view additional integrations.
Is there a clearer way to indicate that more integrations exist beyond the four mentioned?

For example, should we add another panel showcasing a set of other integration logos instead of relying on the button?